Form-Controls/index.html

```diff
@@ -13,15 +13,46 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <div>
+          <label>Name</label>
+          <input type="text" name="name" id="name" required minlength="2" title="Name must be at least 2 characters">
+
+
+        </div>
+        <br>
+        <div>
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" required>
+        </div>
+        <br>
+        <div>
+          <label for="colors">Choose a color:</label>
+          <select name="colors" id="colors" required>
+            <option value="blue">Blue</option>
+            <option value="black">Black</option>
+            <option value="green">Green</option>
+          </select>
+        </div>
+        <br>
+        <div>
+          <label for="sizes">Choose your sizes:</label>
+          <select name="sizes" id="sizes" required>
+            <option value="xs">Xs</option>
+            <option value="s">S</option>
+            <option value="m">M</option>
+            <option value="l">L</option>
+            <option value="xl">Xl</option>
+            <option value="xx">XXL</option>
+          </select>
+        </div>
+        <button>Submit</button>
+
+       
       </form>
     </main>
     <footer>
       <!-- change to your name-->
```

Comment:
Comments are valuable if they help future developers understand a complex piece of code, or a business logic that isn't easily understood by reviewing the code.
In this case, it's probably best to remove the comment!